ow_lander/scripts/all_action_trajectories.py

```diff
@@ -785,17 +786,17 @@ def guarded_move_plan(move_arm, robot, moveit_fk, args):
     return guarded_move_traj, estimated_time_ratio
 
 
-def deliver_sample(move_arm, robot, moveit_fk, args):
+def discard_sample(move_arm, robot, moveit_fk, args):
     """
     :type move_arm: class 'moveit_commander.move_group.MoveGroupCommander'
     :type args: List[bool, float, float, float]
     """
     move_arm.set_planner_id("RRTstar")
```

Comment:
Why do you need to change the current planner?
also once you are done shouldn't you restore the previous planner?

Reply:
i dont need to change the planner. now that the default is rrtstar... removed it